tests/api_model_operations.rs

```diff
@@ -472,12 +472,10 @@ async fn test_delete_noop() {
     let resp: DeleteModelResponse = test_server
         .get_response(
             "default.model.del.my-example-app",
-            serde_json::to_vec(&DeleteModelRequest {
-                version: None,
-            }).unwrap(),
+            serde_json::to_vec(&DeleteModelRequest { version: None }).unwrap(),
```

Comment:
Note: This changes were just a missing `cargo fmt` that I found while running tests in the flake